### PR TITLE
#164069174 Update travis to run tests

### PR DESCRIPTION
**Pull request by CryceTruly**
**What does this PR do**
This PR integrates Travis ci to run tests

**Description of tasks to be completed**
- Add  a .travis.yml config file

**How can this be tested??**
- Open https://travis-ci.com/andela/ah-aquaman-frontend in your browser to view the Travis build

**Any background context you want to provide**
  N/A

#### Applicable screenshots
Screenshot 2019-04-03 at 13 25 44

**What are the relevant pivotal tracker stories**
- [#164069174](https://www.pivotaltracker.com/story/show/164069174)

### CODE REVIEW

File: coverage/clover.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (wasibani-roy) [inline coverage/clover.xml]: Do we need to have the coverage folder pushed to the repo and what would its purpose be??

**Review by wasibani-roy — CHANGES_REQUESTED** [review]

File: coverage/clover.xml (hunk 1 above)
Comment (CryceTruly) [reply coverage/clover.xml]: Thanks, @wasibani-roy for the highlight, I have removed it from version control.

**Review by joelethan — APPROVED** [review]
Thank you for the good work @CryceTruly

**Review by wasibani-roy — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "stable"
+cache:
+  directories:
+  - node_modules
+script:
+  - npm run travis
```
Comment (AmosWels) [inline .travis.yml:8]: add scripts to run tests as well. `npm run test`.

**Review by AmosWels — COMMENTED** [review]
Goodwork @CryceTruly , I hope you'll modify this file after the coveralls as been added. Such that there is an after script as well in the travis file.

How come the build is unknown on your branch?

File: .travis.yml (hunk 2 above)
Comment (CryceTruly) [reply .travis.yml:8]: Hey @AmosWels,`npm run travis` basically runs `npm run test` but not in watch mode
`npm run test` runs the tests but keeps the build hanging
After the 10th-minute Travis treats it as an errored build so we resorted to wring a custom script.

Comment (CryceTruly) [reply .travis.yml:8]: The build passes on the branch however travis looks at the default branch which is develop

**Pull request merged**